source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -128,7 +128,7 @@ documents match.
 
     The ``updateOne()`` and ``updateMany()`` methods cannot make changes
     to a document that violate unique index constraints on the
-    collection. See the MongoDB server manual for more information on
+    collection. See the MongoDB Server manual for more information on
     :manual:`unique indexes </core/index-unique/>`. 
```

Comment:
[suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

```suggestion
    collection. For more information about unique index constraints, see 
    :manual:`unique indexes </core/index-unique/>` in the {+mdb-server+} manual. 
```

Reply:
Sure. Wasn't looking at things like that for this ticket, but happy to change.